.nvmrc

```diff
@@ -0,0 +1 @@
+v18.20.4
```

Comment:
I think we don't need to check this in, right?

Reply:
this is not a check, it is more a convenience if you manage your node version with `nvm`, having this file with the node version you can simply `nvm use` and switch to the expected node version.
It should not hurt to keep it in here, it only requires an update when the `engines` section changes
```json
engines
    "node": "~18.20.3"
  },
```

I can remove it if it should not be in the code repository.